daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -188,7 +188,10 @@ object Converter {
     } yield record(
       contractIdTy,
       ("getAnyChoice", SAny(choice.argBinder._2, translated)),
-      ("getAnyChoiceTemplateTypeRep", fromIdentifier(toApiIdentifier(templateId))),
+      (
+        "getAnyChoiceTemplateTypeRep",
+        fromTemplateTypeRep(toApiIdentifier(templateId)),
+      ),
```

Comment:
The encoding here was wrong, which I only noticed when I tried to inspect the second field of an `AnyChoice` and got a runtime error like `expected SRecord but got STypeRep(...)`, because `TemplateTypeRep` is normally represented as a single field record, unlike `TypeRep` which corresponds to `STypeRep` in LF.

Reply:
This is one more argument against our current approach for translating `Daml.Script` values to LF and back @remyhaemmerle-da